### PR TITLE
Use wait-for-it instead of wait

### PR DESCRIPTION
**Pull request by danquah**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ shared: &test
           export GOSS_FILES_STRATEGY=cp
           # Sleep a bit to ensure that PHP-FPM has time to start up.
           GOSS_SLEEP=15 dgoss run -e GOSS_PHP_VERSION="${PHP_VERSION}" docker-drupal-php7-fpm:sut
+          # Ensure we have wait-for-it and that it works
+          wait-for-it -t 1 localhost:9000
```
Comment (kasperg) [inline .circleci/config.yml]: Could/should this be a part of the goss test?

Comment (danquah) [reply .circleci/config.yml]: Agree!

**Review by kasperg — APPROVED** [review]
:+1: LGTM

**Comment by danquah** [comment]
wohoo

**Pull request merged**